backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -594,55 +594,19 @@ public Project SetStatus(Project project, Workflow workflow)
                 case ("AP-EXE"): // Approve for ERP Exemption
                     this.User.ThrowIfNotAuthorized(Permissions.DisposeApprove, "User does not have permission to approve project.");
                     originalProject.ApprovedOn = DateTime.UtcNow;
-                    originalProject.Properties.ForEach(p =>
-                    {
-                        if (p.BuildingId.HasValue)
-                        {
-                            p.Building.IsVisibleToOtherAgencies = true;
-                            this.Context.Buildings.Update(p.Building);
-                        }
-                        else if (p.ParcelId.HasValue)
-                        {
-                            p.Parcel.IsVisibleToOtherAgencies = true;
-                            this.Context.Parcels.Update(p.Parcel);
-                        }
-                    });
+                    this.Context.SetProjectPropertiesVisiblity(originalProject, true);
```

Comment:
I don't think this is actually valid.  This status is for approving an exemption.  At this point the project itself isn't approved.  Secondly it would be rare for exemption to go to ERP.  If they do, this logic would occur when `AP-ERP` occurred.

Reply:
Should I make this change? @Fosol